internal/app/organisation_store.go

```diff
@@ -44,8 +49,9 @@ func (s *organisationStore) Create(ctx context.Context, name string) (*actor.Org
 	}
 
 	member := &actor.Member{
-		PK:        memberKey(data.SessionID),
-		SK:        organisationKey(organisationID),
+		PK:        organisationKey(organisationID),
+		SK:        memberKey(data.SessionID),
+		Email:     data.Email,
```

Comment:
should we be updating this if it has changed when they login?

Reply:
Agreed we are updating on login for supporters but will discuss with product if we care about GOL email changes.